2DRove/Assets/Scripts/StateMachines/SpitterFSM/SpitterStateManager.cs

```diff
@@ -4,34 +4,82 @@
 
 public class SpitterStateManager : MonoBehaviour
 {
-    // Start is called before the first frame update
-    SpitterBaseState currentState; //
+    SpitterBaseState currentState;
+    public Animator animator;
```

Comment:
I would like for there to be some consistency between finite statemachines, where we can see the attack range of our character. That's what the gizmo is used for at the bottom of this file. In this case, we will be using the Circle for this character, which means you only need one gizmo. If you need help with this, please let me know.

Reply:
Alright, I'll leave it up to your judgement.